storage/aptosdb/src/pruner/ledger_store/ledger_store_pruner.rs

```diff
@@ -147,6 +158,7 @@ impl LedgerPruner {
         self.event_store_pruner
             .prune(db_batch, min_readable_version, current_target_version)?;
 
+        println!("min_readable_version: {}", current_target_version);
```

Comment:
nit: remove this line or change to log?

Reply:
Sorry forgot to delete after the debug was done.